### ISSUE TITLE
Join mp3 files "no items match your search"

### ISSUE THREAD

**Issue by JetzinCat** [openaudible/openaudible#693]
I have some old mp3 audio books that are split into chapters. I'd like to join them without loosing the chapter data. The join function can't find them though. I go to File, Join Audio Files, navigate to the folder and it says "No items match your search".

Is this something OpenAudible doesn't do? From reading the documentation and closed issues it sounds like it should work.

Thank you

Grace

**Comment by openaudible** [comment]
I don't recognize that error message. Feel free to copy and paste from the console. (CTRL-D) Maybe that is from ffmpeg and it doesn't like your audio files for some reason? (Or there are single quotes in the file names?)

But yes! As long as a folder contains more than 1 mp3 or m4b files (and optionally, an image file..) it should work.

you can also try the beta which we improved joining on... with the beta, you can simply drag and drop a folder containing a split book at it will automatically be joined and added to your library. https://openaudible.org/beta

**Comment by JetzinCat** [comment]
Thank you for your quick response. It was enough to get me on the right track and I've worked it out. 

The files were named after each chapter. I've renamed them to the book title - Villette 01.mp3, Villette 02.mp3 ect. It works now.

For anyone else needing to do this, I renamed the whole list as a batch but had to add the 0 before the single digits, otherwise they joined in the order of 1, 11, 12... 2, 21, 22... etc.
